Makefile

```diff
@@ -3,6 +3,12 @@
 ##
 
 SYNC_DATABASE_URL = 'mysql://sample_user:sample_password@localhost/syncstorage_rs'
+PATH_TO_SYNC_SPANNER_KEYS = `pwd`service-account.json
+
+# TODO: replace with rust grpc alternative when ready
```

Comment:
Is there a better one for me to link to than the one in `server-syncstorage` since that repo is going away?

Reply:
might want to insert a `/` before `services-account.json`. Platforms like Linux don't automatically add a path separator to pwd.

Reply:
Also, the google key files could be anywhere. I think that google suggests a common `~/keys` directory or something. I think having one under a given project as a default may be good. If a given user has a known key dir, they can specify it as an env that make will pick up. We could add a comment to the Makefile indicating that.